examples/pigweed-app/nrfconnect/main/main.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <array>
```

Comment:
Not all CHIP files follow that convention, but it makes sense to order includes starting with project headers through CHIP headers through NCS and standard headers. That way, we enforce that all project headers include explicitly everything they need.

Reply:
Actually the std headers turned out to be unnecessary, but anyway I reordered the rest of headers following your instruction.